### ISSUE TITLE
How to get the picture class_name?

### ISSUE THREAD

**Issue by ZengyuanYu** [lars76/object-localization#2]
Thanks your code,it is very cool.But now I want get pic cat or dog localization and classname,what should I do?

**Comment by lars76** [comment]
Hi,

when you want to localize the dog/cat and also want to distinguish between the two classes, you have a mix of regression and classification. 

This means:
1. Add to the csv file a column "class" and load it in train_model.py
2. the CNN has to output now 5 values (4 coords + 0 or 1 for class)
3. You might have to use a merge layer https://keras.io/layers/merge/#concatenate_1 (one branch gets softmax, one branch gets identity activation i.e. "linear")
4. Write a custom loss function that calculates MSE and cross entropy. The coord loss will have a stronger influence, maybe MSE will work for the class too. You could also weight the losses differently.

**Comment by ZengyuanYu** [comment]
Thank you for  your advice, it is good for me.Good day!

**Comment by lars76** [comment]
I just pushed some code that implements this (example_2).
